lib/EntryOptionPlugin.js

```diff
@@ -20,7 +20,7 @@ module.exports = class EntryOptionPlugin {
 		compiler.plugin("entry-option", (context, entry) => {
 			if(typeof entry === "string" || Array.isArray(entry)) {
 				compiler.apply(itemToPlugin(context, entry, "main"));
-			} else if(typeof entry === "object") {
+			} else if(entry && typeof entry === "object") {
```

Comment:
isn't initializing entry with null will be better solution?
(context, entry = null)